cluster-service/deploy/integration/cluster-service-namespace.yaml

```diff
@@ -27,12 +33,12 @@ objects:
     metadata:
       name: namespace-admin
     rules:
-    - apiGroups:
-      - ""
-      resources:
-      - namespaces
-      verbs:
-      - "*"
+      - apiGroups:
+          - ""
+        resources:
+          - namespaces
```

Comment:
If I understand correctly, this grants performing any actions against K8s namespace resources. Is that what we intend?

Reply:
Yes, we want to get, list and delete permissions. So I used the ClusterRole which was already there in the template.